blazor/ai-assistview/speech/text-to-speech.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  Text-to-Speech integration with Blazor AI AssistView Component | Syncfusion
+description: Checkout and learn about Text-to-Speech integration with Blazor AI AssistView component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: AI AssistView
+documentation: ug
+---
+
+# Integration of Text-to-Speech with Azure OpenAI in AI AssistView
```

Comment:
Use the heading **Text-to-Speech** alone

Can you add the single md file like Speech? 

Add the sub heading 

Text-to-Speech

Speech-to-Text

Reply:
Updated headings as per comments.